packages/moonstone/styles/variables.less

```diff
@@ -493,8 +493,7 @@
 @moon-video-slider-tooltip-thumbnail-border-width: 3px;
 @moon-video-slider-tooltip-thumbnail-border-radius: @moon-contextual-popup-border-radius;
 @moon-video-feedback-icon-width:                   0.5em; // The width of the icon is half of its own font size
-@moon-video-feedback-icon-font-size:               3em; // For most icons; The icon size is 3 times the nearby text font size, so the icon is big enough to visually match
-@moon-video-feedback-icon-shrunken-size:           2em; // For play and pause icons, the em value is smaller since these glyphs are larger scale than the others.
+@moon-video-feedback-icon-font-size:               2em;
```

Comment:
I don't believe we want to remove any variables at this point.  I will mark this deprecated.